app/src/main/java/com/framgia/soundclound/service/MusicService.java

```diff
@@ -0,0 +1,280 @@
+package com.framgia.soundclound.service;
+
+import android.annotation.SuppressLint;
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.media.AudioManager;
+import android.media.MediaPlayer;
+import android.os.Binder;
+import android.os.IBinder;
+import android.os.PowerManager;
+
+import com.framgia.soundclound.BaseMediaPlayer;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.screen.playtrack.PlayTrackActivity;
+import com.framgia.soundclound.util.Constant;
+
+import java.util.List;
+import java.util.Random;
+
+/*
+   service
+ */
+
+public class MusicService extends Service implements
+        MediaPlayer.OnPreparedListener, MediaPlayer.OnErrorListener,
+        MediaPlayer.OnCompletionListener, BaseMediaPlayer {
+
+    private static final String LOG_TAG = "fuck";
+    private MediaPlayer player;
+    private List<Track> mTracks;
+    private int curentTrack;
+    private final IBinder musicBind = new MusicBinder();
+    private String songTitle = "";
+    private boolean isShuffle = false;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isShuffle has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName